Base/usr/share/man/man7/Shell-vars.md

```diff
@@ -17,7 +17,17 @@ The value of this variable is used to join lists or split strings into lists, it
 
 2. History
 
-`HISTFILE` (environment)
+`HISTCONTROL` (environment)
+
+The value of this variable is used to determine which entries are kept in the Shell's history, both regarding the current active session and when writing the history to disk on exit. It is unset by default, meaning no entries will be excluded from history.
```

Comment:
> it is unset by default

but it being unset isn't documented...

Reply:
> meaning no entries will be excluded from history

isn't that enough documentation for this case? :smile:

Reply:
I'd just move the "meaning no entries..." to the list below under 'unset'.
I read that part when reading through and it didn't register at all...

Reply:
cool, should be more clear now.

Reply:
It is, thanks!